vocabularies/voc4cat/0007016.ttl

```diff
@@ -0,0 +1,14 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+<https://w3id.org/nfdi4cat/voc4cat_0007016>
+    a skos:Concept ;
+    dcterms:identifier "voc4cat_0007016"^^xsd:token ;
+    skos:definition "The main technique applied to make the catalyst material out of pure substances, e.g. precipitation, impregnation, hydrothermal synthesis, calcination,... "@en ;
```

Comment:
- As before we might not need to mention the examples in the definition. Maybe it is better to add precipitation, impregnation, hydrothermal synthesis, calcination etc as separate concepts and add them as narrower concepts to the "preparation method" concept.